index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="./stylesheet/style.css">
+</head>
+
+<body>
+
+    <nav>
+        <h4>David Kasilia Logo</h4>
+        <img class="menu" src="./images/menu_icon-removebg-preview.png" alt="Menu Icon">
+    </nav>
+
+    <header>
+        <h1>David Kasilia</h1>
+    </header>
+
+
+    <main>
+    <article>
+        <h2>Software Developer</h2>
+    </article>
+  
+
+<div class="text">
+    <p>I am a software developer motivated with passion to work and provide the best product that will impress my clients.
+    My services include building mobile applications and websites. Explore My projects and if you like them feel free to contact me.</p>
+    </div>
+    <img class="line" src="./images/line-1.png" alt="Horizontal line">
```

Comment:
- [ ] Please fix the margin and the padding of the website 
![image](https://user-images.githubusercontent.com/74744787/176555086-45f6d0ca-313a-47ea-a046-f05ff60b1e92.png)